src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.props

```diff
@@ -19,6 +19,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <DebugType Condition="'$(DebugType)' == ''">pdbonly</DebugType>
     <PreserveCompilationContext Condition="'$(PreserveCompilationContext)' == ''">true</PreserveCompilationContext>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
Discussed with Mike offline. The only scenario where this property will force override is if this gets set from the NET Sdk. Since this is not a case, we are fine not having a condition for this.